x/merkledb/db.go

```diff
@@ -244,8 +244,13 @@ type merkleDB struct {
 }
 
 // New returns a new merkle database.
-func New(ctx context.Context, db database.Database, config Config) (MerkleDB, error) {
-	metrics, err := newMetrics("merkledb", config.Reg)
+func New(
+	ctx context.Context,
+	db database.Database,
+	config Config,
+	namespace string,
```

Comment:
Should this namespace be put in the config rather than passed alongside it?

Reply:
I think my rationale for not putting this in the `Config` is that this is probably something we want to explicitly configure and not depend on a default for...